javascript/javascript1/week1/housePricerApp.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <script>
+
+        //HouseyPrice (A House price estimator)
+
+
+
+function costOfHouse() {
+  let houseWide = prompt("What is the width of your house?");
+  let houseDeep = prompt("What is the depth of your house?");
+  let houseHigh = prompt("What is the height of your house?");
+  let gardenSizeInM2 = prompt("What is your garden size?");
+  let volumeInMeters = houseWide * houseDeep * houseHigh;
+  let costOfHouse = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
```

Comment:
I would avoid reusing the same name of your function for a variable within the function.

Reply:
okey, thanks